tests/test.py

```diff
@@ -4350,6 +4351,59 @@ def find_forked_pid(self, filepath):
                     pid = re.search(r'PID: (\d+)', line).group(1)
                     return int(pid)
 
+    def testAcraServerTerminate(self):
+        acra_args = self.get_acra_cli_args({})
+        temp_keystore = self.copy_keystore()
+        config = load_yaml_config('configs/acra-server.yaml')
+        config.update(acra_args)
+        config['keys_dir'] = temp_keystore
+        config['log_to_file'] = tempfile.NamedTemporaryFile().name
+        temp_config = tempfile.NamedTemporaryFile()
+        dump_yaml_config(config, temp_config.name)
+
+        acra = fork(lambda: subprocess.Popen([
+            self.get_acraserver_bin_path(), '--config_file={}'.format(temp_config.name)]))
+        try:
+            self.wait_acraserver_connection(config['incoming_connection_string'])
+        except:
+            stop_process(acra)
+            shutil.rmtree(temp_keystore)
```

Comment:
let's remove `log_to_file` and `temp_config` that are temporary files. will be better to use them as context managers to automatically delete at the end:
```
with NamedTemporaryFile() as log_file, NamedTemporaryFile() as config_file:
  _ = log_file.name
  _ = config_file.name
```
In such case they [will be deleted at the end](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile)